config/l5-swagger.php

```diff
@@ -44,7 +44,7 @@
         |--------------------------------------------------------------------------
          */
         'middleware' => [
-            'api' => ['web', 'auth'],
+            'api' => [],
```

Comment:
This removes auth from the Swagger HTML GUI that documents our REST APIs. The APIs themselves still require auth.